lib/cadet_web/controllers/grading_controller.ex

```diff
@@ -74,12 +88,16 @@ defmodule CadetWeb.GradingController do
   swagger_path :index do
     get("/grading")
 
-    summary("Get a list of all submissions with current user as the grader. ")
+    summary("Get a list of all submissions")
 
     security([%{JWT: []}])
 
     produces("application/json")
 
+    parameters do
+      group(:body, :boolean, "Boolean to show group under grader", required: false)
```

Comment:
`"Show only students in the grader's group when true"`

Reply:
Done